sources/libc/ansi/assert.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+/* --- 7.2 Diagnostics ----------------------------------------------- */ 
+
+NDEBUG
+
+#define assert(ignore) ((void)0)
+
+static_assert
+
+/* --- 7.2.1 Program diagnostics ----------------------------------------------- */ 
+
+void assert(scalar expression);
```

Comment:
This is not valid C

Reply:
ok sorry i will see how to change it

Reply:
I add some changes, it is ok now @sleepy-monax ?